ncm-resolver/src/main/pan/components/resolver/config.pan

```diff
@@ -5,5 +5,12 @@
 
 unique template components/${project.artifactId}/config;
 
-include { 'components/${project.artifactId}/config-common' };
-include { 'components/${project.artifactId}/config-rpm' };
+include 'components/${project.artifactId}/schema';
+include 'pan/functions';
+
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+prefix '/software/components/${project.artifactId}';
+
+'active' ?= true;
+'dispatch' ?= true;
```

Comment:
@ajf8 i was not suggesting you forgot it, but while you are doing some (much appreciated) cleanup, why not fix it.